web/src/js/components/Dashboard/DashboardComponent.js

```diff
@@ -441,7 +441,7 @@ class Dashboard extends React.Component {
 
     const isYahooUser = user && user.id === YAHOO_USER_ID
 
-    const notificationsToShow = this.state.notificationsToShow
+    // const notificationsToShow = this.state.notificationsToShow
```

Comment:
Looks like this might be an accident?

Reply:
@jedtan This should have been commented out before. It is an unused var at the moment. My editor gave me a warning.